src/lightning/fabric/fabric.py

```diff
@@ -799,5 +799,13 @@ def _is_using_cli() -> bool:
     return bool(int(os.environ.get("LT_CLI_USED", "0")))
 
 
+def is_wrapped(module: Union[nn.Module, _FabricModule]) -> bool:
+    """Method to check if nn.Module is already wrapped as a FabricModule
+    Args:
+        module: The module to be checked if it is _FabricModule or not
```

Comment:
Can we extend this to an arbitrary object and not only check for _FabricModule, but also for _FabricOptimizer etc.?
That would be kind of consistent with `_unwrap_objects`

Reply:
@awaelchli, would you like? function for each check ??

Reply:
No, I think just extending the check inside this function is the best :))